dist/images/ovnkube.sh

```diff
@@ -1030,14 +1059,15 @@ ovn-node() {
     ${ovn_unprivileged_flag} \
     --nodeport \
     --mtu=${mtu} \
-    ${OVN_ENCAP_IP} \
+    ${ovn_encap_ip_flag} \
     --loglevel=${ovnkube_loglevel} \
     --loglevel=${ovnkube_loglevel} \
     --logfile-maxsize=${ovnkube_logfile_maxsize} \
     --logfile-maxbackups=${ovnkube_logfile_maxbackups} \
     --logfile-maxage=${ovnkube_logfile_maxage} \
     ${hybrid_overlay_flags} \
     ${disable_snat_multiple_gws_flag} \
+    --k8s-cacert=${K8S_CACERT} \
```

Comment:
same question

Reply:
When ovnkube node running on Smart-NIC, it requires K8S token and CA cert to access K8s API.
these need to be provided to ovnkube node.

looking more closely, we should provide both the same way, either through CLI or Env variables.
I will remove this so both will be taken from Environment variables for consistency.
(with current code, one is taken from ENV variable and the other from CLI flag)